.github/workflows/nodejs.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v3
-        with:
```

Comment:
I'm confused, what happened here for this to build? Did you do something in AWS Amplify?

Reply:
yeah, would be nice to know what was the issue, so we can avoid it in the future.

Will approve since all looks good and build working.

Reply:
@karelianpie the amplify builder needs to have access to the github npm package registry for installing libs.
I've added this line to our amplify build settings to make the build succeed (needs to be done on all repos) :
`- echo "//npm.pkg.github.com/:_authToken=${GITHUB_NPM_TOKEN}" >> .npmrc`

Reply:
Thanks @mariuspod! That makes sense 🙇